runtime/sema/type.go

```diff
@@ -155,7 +156,7 @@ type Type interface {
 	//
 	Unify(
 		other Type,
-		typeParameters *TypeParameterTypeOrderedMap,
+		typeParameters *orderedmap.OrderedMap[*TypeParameter, Type],
```

Comment:
similarly, maybe it's worth defining type aliases for these commonly used map types.

Reply:
Added in [9045c19](https://github.com/onflow/cadence/pull/1804/commits/9045c1981258031fe0f9946dea9c4963b44d7e9d)